SPECS/openssh/openssh.spec

```diff
@@ -32,6 +32,9 @@ Patch306:       pam_ssh_agent_auth-0.10.2-compat.patch
 # Fix NULL dereference from getpwuid() return value
 # https://sourceforge.net/p/pamsshagentauth/bugs/22/
 Patch307:       pam_ssh_agent_auth-0.10.2-dereference.patch
+#CVE Patches
+#This CVE Patches both CVE-2025-26465 and CVE-2025-26466
+Patch400:       CVE-2025-26465.patch
```

Comment:
Does CVE-2025-26466.nopatch need to be included in the spec like Patch100 above? Should the Patch100 line be taken out or commented out since it is unused if nopatch files are not meant to be included?

Reply:
no.  nopatch files do not need to be included in the spec file.